### PR TITLE
ci: update README

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Loong Dai <loong.dai@intel.com>

Update README due to changes in #21767.

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -141,9 +141,9 @@ The `./ci/run_envoy_docker.sh './ci/do_ci.sh <TARGET>'` targets are:
 * `bazel.compile_time_options` &mdash; build Envoy and run tests with various compile-time options toggled to their non-default state, to ensure they still build.
 * `bazel.compile_time_options <test>` &mdash; build Envoy and run a specified test or test dir with various compile-time options toggled to their non-default state, to ensure they still build.
 * `bazel.clang_tidy <files>` &mdash; build and run clang-tidy specified source files, if no files specified, runs against the diff with the last GitHub commit.
-* `check_format`&mdash; run `clang-format` and `buildifier` on entire source tree.
-* `fix_format`&mdash; run and enforce `clang-format` and `buildifier` on entire source tree.
-* `format_pre`&mdash; run validation and linting tools.
+* `check_proto_format`&mdash; run `clang-format` and `buildifier` on entire source tree.
+* `fix_proto_format`&mdash; run and enforce `clang-format` and `buildifier` on entire source tree.
+* `format`&mdash; run validation and linting tools.
```
Comment (phlax) [inline ci/README.md:146]:
```suggestion
* `format`&mdash; run validation, linting and formatting tools.
```

**Review by phlax — CHANGES_REQUESTED** [review]
@daixiang0 thanks for picking this up - i think we need to adjust the descriptions tho

File: ci/README.md [hunk 2]
```diff
@@ -141,9 +141,9 @@ The `./ci/run_envoy_docker.sh './ci/do_ci.sh <TARGET>'` targets are:
 * `bazel.compile_time_options` &mdash; build Envoy and run tests with various compile-time options toggled to their non-default state, to ensure they still build.
 * `bazel.compile_time_options <test>` &mdash; build Envoy and run a specified test or test dir with various compile-time options toggled to their non-default state, to ensure they still build.
 * `bazel.clang_tidy <files>` &mdash; build and run clang-tidy specified source files, if no files specified, runs against the diff with the last GitHub commit.
-* `check_format`&mdash; run `clang-format` and `buildifier` on entire source tree.
-* `fix_format`&mdash; run and enforce `clang-format` and `buildifier` on entire source tree.
-* `format_pre`&mdash; run validation and linting tools.
+* `check_proto_format`&mdash; run `clang-format` and `buildifier` on entire source tree.
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply ci/README.md:144]: Updated.

File: ci/README.md [hunk 3]
```diff
@@ -141,9 +141,9 @@ The `./ci/run_envoy_docker.sh './ci/do_ci.sh <TARGET>'` targets are:
...
+* `fix_proto_format`&mdash; run and enforce `clang-format` and `buildifier` on entire source tree.
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply ci/README.md:145]: Updated.

File: ci/README.md (hunk 1 above)
Comment (daixiang0) [reply ci/README.md]: Updated.

**Comment by daixiang0** [comment]
friendly ping @phlax

**Review by phlax — APPROVED** [review]
lgtm, thanks @daixiang0

**Pull request merged**